test/unit/module/algo/algorithm/sums_special_cases.cpp

```diff
@@ -143,9 +143,9 @@ TTS_CASE("eve.algo.transform_reduce, stateful map/reduce")
             eve::algo::transform_reduce(
                 v,
                 [coeff](auto x) { return x * coeff; },
-                std::pair {[_ = coeff](auto x, auto y)
+                std::pair {[state = coeff](auto x, auto y)
                            {
-                             (void)_;
+                             (void)state;
```

Comment:
This is reasonable, I don't know about tolerances increase.

Reply:
1 ULPs sounds fait on those large functions. I'll cross check with jtlap

Reply:
@DenisYaroshevskiy @jtlap says it is fine, so I am gonna approve if it is OK for you.